deluge/log.py

```diff
@@ -107,7 +107,7 @@ def findCaller(self):  # NOQA
 }
 
 
-def setup_logger(level="error", filename=None, filemode="w"):
+def setup_logger(level="error", filename=None, filemode="w", max_size="50M"):
```

Comment:
Having a human readable has the advantage that it is human readable. Having int in megabytes means you cannot set log size smaller than 1MB.